packages/block-editor/src/components/keyboard-shortcuts/README.md

```diff
@@ -0,0 +1,35 @@
+# Keyboard Shortcuts
+
+The `KeyboardShortcuts` component handles the operation and behavior of keyboard shortcuts in the block editor.
+
+![Keyboard shortcuts examples](https://make.wordpress.org/core/files/2020/09/keyboard-shortcuts-examples.png)
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+## Development guidelines
+
+### Usage
+
+Displays a block toolbar for a selected block.
```

Comment:
This description is incorrect. As far as I can tell, this component registers all standard shortcuts that can be used inside the block editor. @youknowriad, can you confirm?